src/constants/ukef-id.constant.ts

```diff
@@ -1,10 +1,18 @@
 export const UKEFID = {
   // Prefix is used for 10 digit id (Deal, Facility) and 8 digit id (Party/Customer)
-  MAIN_ID_PREFIX: {
-    DEV: '0030',
-    QA: '0040',
-    PROD: '0020',
+  MAIN_ID: {
+    PREFIX: {
+      DEV: '0030',
+      QA: '0040',
+      PROD: '0020',
+    },
+    TEN_DIGIT_REGEX: /^00\d{8}$/,
+    TEN_DIGIT_REGEX_STRING: '/^00\\d{8}$/',
```

Comment:
Do we need string version of pattern?
For me api-tests are working with regex object